solutions/USACO dec15-feast-gold.mdx

```diff
@@ -47,3 +47,70 @@ int main() {
 </CPPSection>
 
 </LanguageSection>
+
+## Solution 2 - Using BFS
+
+<LanguageSection>
+
+<CPPSection>
+
+```cpp
+#include <bits/stdc++.h>
+using namespace std;
+
+const int maxn = 5e6 + 5;
+bool pass[maxn];
+
+struct node{
+    int val; bool drunk;
+	

```

Comment:
Could you please remove the gaps between the lines? Those empty lines are redundant.